packages/synapse-react-client/src/components/TanStackTable/ColumnHeaderEnumFilter.test.tsx

```diff
@@ -14,52 +17,89 @@ type TestTableRow = {
   multiSelectWithCounts: string
   singleSelect: string
   numeric: number
+  defaultClientSideFacetCalculation: string
 }
 
 const columnHelper = createColumnHelper<TestTableRow>()
 
+function getFacetedUniqueValues(table: Table<TestTableRow>, columnId: string) {
```

Comment:
Update test for component that creates facet filter UI using TanStack table data